src/blocks/Sidebar/style.css

```
@@ -0,0 +1,209 @@
+    margin-bottom: var(--tree-item-spacing, 4px);
+}
+
+.wedocs-section-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    cursor: pointer;
+    padding: 8px 12px;
+    background-color: var(--tree-header-bg, transparent);
+    color: var(--tree-header-text, inherit);
+    padding: var(--tree-header-padding, 8px 12px);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)